src/test/java/de/wi2020sebgroup1/cinema/controller/AccountControllerTest.java

```diff
@@ -104,6 +104,7 @@ User getUser() {
     	return l;
     }
     
+    /* Funktioniert nicht
```

Comment:
Sollte gehen, wenn immer ein anderer User erstellt wird. Wenn die Tests nicht funktionieren ist es halt schwierig wenn die nur auskommentiert werden, weil ich deine Änderung zB nicht im Detail kenne und die Tests neu überlegen muss. Hier wirds denke ich dran liegen dass der versucht mehrfach neue User mit DrBackmischung und mathis.neunzig@gmail.com anzulegen in einem Gang

Reply:
Ich probiere es zu fixen, aber ich hab halt leider auch keine Ahnung wie deine Tests funktionieren :/